src/pages/ConfidentialityPage/components/Confidentiality/ConfidentialityPart.tsx

```diff
@@ -1,7 +1,103 @@
+          <p>
+            <span>
+              Şəxsi məlumat deyərkən müştərilərin Sendex MMC-nin xidmətlərindən
+              istifadə zamanı şirkətə təqdim etdiyi bütün məlumatlar nəzərdə
+              tutulur.
+            </span>
+          </p>
+          <p>&nbsp;</p>
+          <p>
+            <span>ŞƏXSİ MƏLUMATLARIN QORUNMA PRİNSİPLƏRİ</span>
+          </p>
+          <p>
+            <span>Ümumi prinsiplər:</span>
+          </p>
+          <p>
+            <span>
+              Sendex MMC aşağıda qeyd edilən məqsədlərlə müştərinin &nbsp;şəxsi
+              məlumatlarını istifadə edə və ya açıqlaya bilər:
+            </span>
+          </p>
+          <p>
+            <span>- Xidmət seçimlərinizi təyin etmək üçün</span>
+          </p>
+          <p>
+            <span>
+              - İnformasiya, təbliğat, məhsul və xidmətlərimiz və ya reklam
+              materiallarını sizə göndərmək üçün
+            </span>
+          </p>
+          <p>
+            <span>- Sendex MMC-nin maraqlarını qorumaq üçün</span>
+          </p>
+          <p>
+            <span>- Ədalət mühakiməsi üçün</span>
+          </p>
+          <p>
+            <span>
+              - Sizin maraqlarınıza uyğun olduğunu düşündüyümüz, Sendex MMC
+              xidmətləri və ya üçüncü tərəfin xüsusi təkliflərini təbliğ etmək
+              üçün
+            </span>
+          </p>
+          <p>&nbsp;</p>
+          <p>
+            <span>
+              Sendex MMC qeydiyyat formasında tələb edilən şəxsi məlumatları
+              Azərbaycan Respublikası Dövlət Gömrük Komitəsi tələblərinə və
+              ehtiyaclarınıza uyğun daha yaxşı xidmət göstərmək məqsədi ilə
+              toplayır.
+            </span>
+          </p>
+          <p>
+            <span>Təhlükəsizlik prinsipləri:</span>
+          </p>
+          <p>
+            <span>
+              Sendex MMC şəxsi məlumatların səhvən silinməsi və ya açıqlanmaması
+              üçün Şəxsi Məlumatların gizliliyi və təhlükəsizliyini qorumaq
+              məqsədilə təşkilati və texniki addımlar da daxil olmaqla bütün
+              ehtiyat tədbirlərinin görülməsinə məsuliyyət daşıyır.
+            </span>
+          </p>
+          <p>
+            <span>
+              Müştəri Qeydiyyat Formasında tələb edilən məlumatlara verilən
+              cavablara görə məsuliyyət daşıyır. Əgər Qeydiyyat formasında tələb
+              edilən məlumatlara natamam və ya qeyri-dəqiq cavab verilsə, bu,
+              müştəriyə göstərilən xidmətin dayandırılması və ya xidmətin
+              tamamilə sonlandırılması ilə nəticələnə bilər.
+            </span>
+          </p>
+          <p>&nbsp;</p>
+          <p>
+            <span>
+              Sendex MMC müştərilərinin şəxsi məlumatlarının qorunmasına
+              cavabdehdir.
+            </span>
+          </p>
+          <br />
+          <br />
+          <br />
+          <br />
+          <br />
+          <br />
+          <br />
+          <br />
```

Comment:
why do you need these br-tags ?)))
if you need some whitespace, it might be margin =)